packages/components/forma-36-react-timepicker/src/Timepicker.tsx

```diff
@@ -0,0 +1,377 @@
+      }
+    },
+    [onChange, date]
+  );
+
+  const handleKeyUp = useCallback(
+    event => {
+      if (isHotkey('enter', event)) {
+        setTimeSuggestionOpen(false);
+      }
+
+      const activeIndex = filteredHours.findIndex(elem => elem.isActive);
+      let nextIndex = 0;
+
+      if (isHotkey('arrowUp', event) && filteredHours[activeIndex + 1]) {
+        nextIndex =
+          activeIndex - 1 > 0 ? activeIndex - 1 : filteredHours.length - 1;
+        handleChange(filteredHours[nextIndex].format12H);
+      }
+
+      if (isHotkey('arrowDown', event)) {
+        nextIndex =
+          activeIndex + 1 < filteredHours.length ? activeIndex + 1 : 0;
+        handleChange(filteredHours[nextIndex].format12H);
+      }
+    },
+    [handleChange, filteredHours]
+  );
+
+  const handleKeyDown = useCallback(event => {
+    if (isHotkey('arrowUp', event) || isHotkey('arrowDown', event)) {
+      event.preventDefault();
+    }
+  }, []);
+
+  const handleFocus = useCallback(e => {
+    e.preventDefault();
+    e.target.select();
+    setTimeSuggestionOpen(true);
+  }, []);
+
+  const handleBlur = useCallback(
+    e => {
+      const time = getTimeFromUserInputOrDefaultToValue();
+      setSelectedTime(time);
+      closeDropdown(e);
+      onBlur();
+    },
+    [
+      getTimeFromUserInputOrDefaultToValue,
+      setSelectedTime,
+      closeDropdown,
+      onBlur,
+    ]
+  );
+
+  return (
+    <div className={styles.timePicker}>
+      {labelText && (
+        <FormLabel required={isRequired} htmlFor="scheduleTimeForm">
+          {labelText}
+        </FormLabel>
+      )}
+      <div className={styles.inputWrapper} id="scheduleTimeForm">
+        <Dropdown
+          className={styles.dropdown}
+          dropdownContainerClassName={styles.dropdownContainer}
+          // TODO: Fix getContainerRef on Dropdown to accept ref object. F36 4.0 Breaking Change
+          // @ts-ignore
+          getContainerRef={setDropdownContainer}
+          toggleElement={
+            <input
+              ref={inputRef}
+              className={styles.timeInput}
+              name="time input"
+              data-test-id="time"
+              value={selectedTime}
+              onKeyUp={handleKeyUp}
+              onKeyDown={handleKeyDown}
+              onFocus={handleFocus}
+              onBlur={handleBlur}
+              onChange={e => handleChange(e.target.value)}
+            />
+          }
+          isOpen={isTimeSuggestionOpen}
+        >
+          <DropdownList maxHeight={200} listRef={listRef}>
+            {filteredHours.map((hour: hour) => {
+              return (
+                <DropdownListItem
+                  testId="time-suggestion"
+                  className={hour.isActive ? styles.selectedTime : undefined}
+                  onClick={() => {
+                    handleChange(hour.format12H);
+                    setTimeSuggestionOpen(false);
+                  }}
+                  key={hour.format12H}
```

Comment:
```suggestion
                  key={hour}
```
shouldn't it be just `hour`?